skills/genomics/genomics-epigenomics/references/output_contract.md

```
@@ -0,0 +1,19 @@
+<!-- Generated by scripts/migrate_skill.py at 2026-05-10T12:44:19Z -->
+<!-- Kind: references/output_contract.md | Source: SKILL.md -->
+
+## Output Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)